service/items.go

```diff
@@ -6,54 +6,51 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/rodrigomkd/go-rest-api/common"
-	"github.com/rodrigomkd/go-rest-api/config"
+	"github.com/rodrigomkd/go-rest-api/model"
+	"github.com/rodrigomkd/go-rest-api/service/api"
+	"github.com/rodrigomkd/go-rest-api/service/csv"
 )
 
-type item struct {
-	ID          int    `json:"ID"`
-	Description string `json:"Description"`
+//struct
+type Service struct {
+	cs         csv.CSVService
+	api        api.ApiService
+	dataSource string
 }
 
-//GetItems - Returns items from CSV
-func GetItems() ([]item, error) {
-	items := []item{}
-
-	csvLines, err := common.ReadCSV(config.ReadConfig().DataSource)
-	if err != nil {
-		log.Println("Error reading CSV: ", err)
-		return items, err
+func New(cs csv.CSVService, api api.ApiService, dataSource string) *Service {
+	return &Service{
+		cs:         cs,
+		api:        api,
+		dataSource: dataSource,
 	}
-	for _, line := range csvLines {
-		id, err := strconv.Atoi(line[0])
-		if err != nil {
-			log.Println("Error ID is not an Integer: ", err)
-			return items, err
-		}
+}
 
-		temp := item{
-			ID:          id,
-			Description: line[1],
-		}
-		items = append(items, temp)
-		log.Println("items: ", items)
-	}
+//GetItems - Returns items from CSV
+func (s Service) GetItems() ([]model.Activity, error) {
+	items := s.api.GetActivities()
```

Comment:
You don't want to replace what you already had to use the api, you had to keep your endpoints that read from the csv file, and have a different one that would call the API, and save to the file, don't mix the 2 flows